binr/ragg2/ragg2.c

```diff
@@ -108,6 +109,257 @@ static int openfile (const char *f, int x) {
 }
 #define ISEXEC (fmt!='r')
 
+static char* parseCFile(const char *file, const char *arch, const char *os, int bits) {
```

Comment:
Put all this new logic in a separate file. Not in ragg2.c and lrobably we want a c api to do that. So having this inside libr/egg woukd be better